staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -206,6 +206,16 @@ var (
 		[]string{"filter"},
 	)
 
+	// requestAbortsTotal is a number of aborted requests with http.ErrAbortHandler
+	requestAbortsTotal = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "apiserver_request_aborts_total",
+			Help:           "Number of requests which apiserver aborted possibly due to a timeout for each group, version, verb, resource, subresource and scope",
```

Comment:
nit: due to a timeout<comma>, for each group ...

Otherwise it reads that it timed out for every group.